### ISSUE TITLE
Remote server rewrite rules accidentally blocking valid directories for apache

### ISSUE THREAD

**Issue by simoncoggins** [totara/totara-docker-dev#207]
The config file apache/config/remote-server.conf includes the lines:

```
# Don't allow the following files/directories to be accessed
RewriteRule (cli|tests|db|lang|classes|amd\/src|rb_sources|yui\/src) - [R=404]
RewriteRule \.(txt|md|json|xml|mustache) - [R=404]
RewriteRule \/\. - [R=404]
```

In a recent feature we added a folder 'totara/api/client/' and this is being caught by the 'cli' part of this rule because it's doing substring matching. This leads to 404 errors when you try to view that page.

We might want to match against full folder names by including a leading and trailing slash in the rule.

**Comment by simoncoggins** [comment]
The proposed fix doesn't seem correct to me - issue was the use of these rules caused problems for remote servers (they were only in the remote-server.conf as it is more risky when exposed publicly. The patch adds the same rules to the local server.conf, and doesn't fix the issue in the remote-server.conf.

**Comment by derschatta** [comment]
sorry @simoncoggins that was sloppy from me. you are right, totally not addressing the problem

**Comment by derschatta** [comment]
fixed in #227
